src/benchmark/tpch_data_micro_benchmark.cpp

```diff
@@ -35,16 +35,14 @@ class TPCHDataMicroBenchmarkFixture : public MicroBenchmarkBasicFixture {
     const auto scale_factor = 10.0f;
     const auto default_encoding = EncodingType::Dictionary;
 
-    auto benchmark_config = BenchmarkConfig::get_default_config();
+    const auto benchmark_config = std::make_shared<BenchmarkConfig>();
     // TODO(anyone): setup benchmark_config with the given default_encoding
     // benchmark_config.encoding_config = EncodingConfig{SegmentEncodingSpec{default_encoding}};
```

Comment:
@Bouncner is this still relevant? AFAIK the default encoding spec is already used by the benchmark config.

Reply:
Good question. Feel free to remove.

Reply:
Ah, it gets kind of clearer with the surrounding code:
https://github.com/hyrise/hyrise/blob/59e469fcdd16e941ebe241406ca9603bac8c1a07/src/benchmark/tpch_data_micro_benchmark.cpp#L36-L44

So it is basically just a helper here. We could either explicitly set the declared default spec via the member of `BenchmarkConfig` or just print the member of the default config. I'd just go for the second version, as someone who touches this and wants to try out a different encoding here should also easily find out how to do that.